.github/workflows/run-tests.yml

```diff
@@ -143,6 +146,13 @@ jobs:
         id: typescript-linting
         if: ${{ always() }}
         run: yarn lint-typescript -f junit -o typescript-test-results/eslint/lint-results.xml
+      - if: ${{ failure() && (steps.typescript-tests.outcome != 'success' || steps.typescript-linting.outcome != 'success') }}
+        uses: ./github-actions/summarize-errors
+        with:
+          location: "${{github.workspace}}/typescript-test-results"
+          project-name: "typescript"
+          artifact-name: typescript-tests-${{ runner.os }}-${{ matrix.java }}-${{ matrix.node.artifact }}
+          framework: "mocha"
```

Comment:
I like your notion of having separate frameworks and handling them together. Hopefully, sometime in near future, we can create something like this for self-evaluation as well. That'll look really cool on public-facing demos.